spec/unit/primitive_consul_health_spec.rb

```diff
@@ -49,4 +49,12 @@
       choregraphie.cleanup.each(&:call)
     end
   end
+
+  context 'when consul is unavailable' do
```

Comment:
maybe you should simply update the existing "passing" test scenario to timeout then return a working value.
Otherwise, you are just showing that failing consul makes choregraphie fails (which is kind of obvious). We are more interested in "temporary hiccup from consul does not fail choregraphie"